must-gather/collection-scripts/gather

```diff
@@ -39,4 +39,7 @@ done
 # Collect sandboxed-containers details
 /usr/bin/gather_sandboxed_containers_logs
 
+# Collect operator details
+/usr/bin/gather_operator
```

Comment:
@pmores, any specific reason for creating a `gather_operator` file instead of re-using the `gather_sandboxed_containers_logs`?

I'd prefer keeping all the "operator related & operator itself" logs as part of the `gather_sandboxed_containers_logs`, at least for now.

Reply:
@fidencio Sure, I didn't understand `gather_sandboxed_containers_logs` was the right place for my stuff, sorry.  I'll remove the new script and move its contents to the existing file.

Reply:
@fidencio I'm implementing your request.  While it's clear now where to put code, I'm struggling a bit trying to find a reasonable path to put the gathered *data* to. The script `gather_sandboxed_containers_logs` generally writes to `/must-gather/sandboxed-containers` where it creates subdirs most of the time.

I could dump the operator data directly into the directory but that might litter it considerably, especially with time.

Or I could create a subdir too, however one with the obvious name (`openshift-sandboxed-containers-operator`) already exists in `/must-gather/sandboxed-containers/namespaces` so it'd be rather confusing to create same-named one in `/must-gather/sandboxed-containers`...

What's the idea behind the data directory layout and where does the data generated by this PR fit?

Reply:
I replied as a review.